<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>149</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COM Port Setup</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_com_rescan">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>87</y>
     <width>149</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>72</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>rescan list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_com_ports">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>27</y>
     <width>149</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>107</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>9</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>117</y>
     <width>149</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>72</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_com_status">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>57</y>
     <width>149</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COM Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
